recipes/wt/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, CMake
+from conan.tools.build import cross_building
+import os
+
+
+class TestPackageV1Conan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not cross_building(self):
+            bin_path = os.path.join("bin", "test_package")
+            args = " --docroot . --http-listen http://127.0.0.1:8080"
+            self.run(bin_path + args, run_environment=True)
```

Comment:
```suggestion
            self.run(bin_path, run_environment=True)
```
No longer need arguments